bin/light-base/src/runtime_service.rs

```diff
@@ -495,6 +495,41 @@ impl<TPlat: Platform> RuntimeService<TPlat> {
         subscription.next().await.unwrap()
     }
 
+    /// Returns the value of the `:code` and `:heap_pages` storage items of the given block.
+    ///
+    /// Returns `None` if the runtime of that block isn't known by the runtime service, for
```

Comment:
looks like this fn always returns `Some(...)`

Reply:
There are two `?`s within the body of the function that can make it return `None`

Reply:
Oh, right. Sorry